src/GraphControl/pch.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+//C4453: A '[WebHostHidden]' type should not be used on the published surface of a public type that is not '[WebHostHidden]' 
+// This warning is disabled because the types in this app will not be published for use in javascript environment
+#pragma warning(disable:4453)
+
+// Windows headers define a min/max macro.
+// Include the below #def to avoid this behavior.
+#define NOMINMAX
+
+#include <collection.h>
+#include <ppltasks.h>
+#include <pplawait.h>
+#include <concrt.h>
+#include <future>
+#include <memory>
+#include <assert.h>
+#include <functional>
+#include <string>
+#include <sstream>
+
+// DirectX headers
+#include <d2d1_3.h>
+#include <d3d11_4.h>
+#include <dwrite_3.h>
+#include <DirectXColors.h>
+#include <wincodec.h>
+#include <wrl.h>
+#include <windows.ui.xaml.media.dxinterop.h>
+
+// C++/WinRT
+#include "winrtHeaders.h"
+
+#include "Control/Grapher.h"
```

Comment:
nit: no project headers in the pch

Reply:
I can't find an explanation for the technical reason, but there needs to be a project header in the pch in-order for the XAML compiler to generate correct type information.  Removing the project header results in:
```
2>s:\repos\calculator\src\graphcontrol\generated files\xamltypeinfo.g.cpp(164): error C2039: 'GraphControl': is not a member of '`global namespace''
```
If you look in the Calculator project, the pch contains `App.xaml.h`.  If you make a blank Visual C++ app, the pch contains 'App.xaml.h'.  If you look at our other apps, it's the same.  I can't find any documentation explaining why..  @jlaanstra, do you know?